jans-cli-tui/cli_tui/plugins/999_jans/main.py

```
@@ -106,3 +129,184 @@ async def coroutine():
 
         asyncio.ensure_future(coroutine())
+
+
+    def validate_ssa(self, ssa, iss):
+
+        # get openid configuration
+        open_id_url = f"https://{iss}/{config_cli.AUTH_DISCOVERY_ENDPOINT}"
+        try:
+            response = config_cli.session.get(
+                url=open_id_url,
+                verify=False if config_cli.args.noverify else True,
```

Ruff (0.15.10)

[warning] Use `not ...` instead of `False if ... else True`

Replace with `not ...`

(SIM211)